migrations/2_deploy_MyToken_1155.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config()
+var BigNumber = require('bignumber.js');
+
+const {TOKEN_BASE_URI, TOKEN_PRICE_BY_ETH, TOKEN_PRICE_BY_ERC20, COUNT_OF_SET, AMOUNT_LIMIT_OF_TOKEN} = process.env;
+const MyToken_1155 = artifacts.require('MyToken_1155');
+
+module.exports = function (deployer) {
```

Comment:
you can expect function 2-nd argument here - `network`, and check if 

> 
```suggestion
module.exports = function (deployer, network) {
    if (network !== 'mainnet') { // deploy your ERC20 and use it's address instead of the address from env variable 
    }
```